support/cas-server-support-duo-core-mfa/src/main/java/org/apereo/cas/adaptors/duo/authn/DuoDirectCredential.java

```diff
@@ -21,11 +22,13 @@
 @Getter
 @AllArgsConstructor
 @EqualsAndHashCode
-public class DuoDirectCredential implements Credential {
+public class DuoDirectCredential implements Credential, MultifactorAuthenticationCredential {
```

Comment:
Shouldn't DuoCredential also implement the interface?

Reply:
It Does